src/Build/BackEnd/BuildManager/BuildParameters.cs

```diff
@@ -770,6 +771,12 @@ public string OutputResultsCacheFile
             set => _outputResultsCacheFile = value;
         }
 
+        public string ResourceManagerSemaphoreName
+        {
+            get => _resourceManagerSemaphoreName;
+            set => _resourceManagerSemaphoreName = value;
+        }
```

Comment:
I wanted to suggest you make this an autoproperty:
```suggestion
        public string ResourceManagerSemaphoreName { get; set; } = $"MSBuild.{Guid.NewGuid().ToString()}";
```

but that wouldn't work with translation 😢